mgmt_cli_test.py

```diff
@@ -349,9 +349,8 @@ def test_manager_sanity(self):
             self.test_mgmt_cluster_crud()
         with self.subTest('Mgmt cluster Health Check'):
             self.test_mgmt_cluster_healthcheck()
-        # test_healthcheck_change_max_timeout requires a multi dc run. And since ipv6 cannot run in multi dc, this test
```

Comment:
good that this comment is going away, cause the ipv6 part of it, is alway wrong. we can use ipv6 on AWS with multi-dc cases.